### DISCUSSION TITLE
Stuck in configuration reboot loop...

### DISCUSSION THREAD

**Discussion by Kur0dani** [sebanc/brunch#1301] [Q&A]
I might not write this in the correct forum, if so please tell me where to go and I'll be on my way.

I tried installing a dual boot Windows 10 / Chrome OS on my ASUS t300 chi transformer laptop. I followed these instructions: https://beebom.com/dual-boot-chrome-os-windows-10/

I used "chromeos_14150.64.0_rammus_recovery_stable-channel_mp-v2" with "brunch_r93_stable_20211002" and the installation went fine. I dual boot using Grub2Win 2.3.1.4 and it seems to be working fine. However, every time I try to start up chrome I go through a bunch of configuration options and by the end it says: 
"Brunch configuration saved. Press any key to reboot your computer..."
I do, the computer reboots and I go back to the configurations again with the same result. I never actually enter Chrome OS. What am I doing wrong? How do I actually start up Chrome OS?

**Comment by ycavan** [comment]
Sounds like you only put in the brunch configure portion in your grub config.  The installer text file actually gives you 2 configs: the top one is the actual brunch startup config & the bottom one is the brunch configure startup config.

**Reply by Kur0dani** [reply]
@ycavan Are you talking about the text from the installer that I copied into Grub2Win custom boot? It is as follows:

```
rmmod tpm
img_part=/dev/sda5
img_path=/chromos.img
search --no-floppy --set=root --file $img_path
loopback loop $img_path
source (loop,12)/efi/boot/settings.cfg
linux (loop,7)/kernel boot=local noresume noswap loglevel=7 options= chromeos_bootsplash= edit_brunch_config=1 \
cros_secure cros_debug loop.max_part=16 img_part=$img_part img_path=$img_path
initrd (loop,7)/lib/firmware/amd-ucode.img (loop,7)/lib/firmware/intel-ucode.img (loop,7)/initramfs.img
```

Is something missing there? Or maybe I shouldn't have used the rammus version? is there a way to switch that or will I have to start over if that's the case?

**Comment by ycavan** [comment]
Here's the txt file I got when I did my install a while ago...

To boot directly from this image file, add the lines between stars to either:
- A brunch usb flashdrive grub config file (then boot from usb and choose boot from disk image in the menu),
- Or your hard disk grub install if you have one (refer to you distro's online resources).
********************************************************************************
menuentry "ChromeOS" --class "brunch" {
	rmmod tpm
	img_part=/dev/nvme1n1p1
	img_path=/chromeos.img
	search --no-floppy --set=root --file $img_path
	loopback loop $img_path
	source (loop,12)/efi/boot/settings.cfg
	if [ -z $verbose ] -o [ $verbose -eq 0 ]; then
		linux (loop,7)$kernel boot=local noresume noswap loglevel=7 options=$options chromeos_bootsplash=$chromeos_bootsplash $cmdline_params \
			cros_secure cros_debug loop.max_part=16 img_part=$img_part img_path=$img_path \
			console= vt.global_cursor_default=0 brunch_bootsplash=$brunch_bootsplash quiet
	else
		linux (loop,7)$kernel boot=local noresume noswap loglevel=7 options=$options chromeos_bootsplash=$chromeos_bootsplash $cmdline_params \
			cros_secure cros_debug loop.max_part=16 img_part=$img_part img_path=$img_path
	fi
	initrd (loop,7)/lib/firmware/amd-ucode.img (loop,7)/lib/firmware/intel-ucode.img (loop,7)/initramfs.img
}

menuentry "ChromeOS (settings)" --class "brunch-settings" {
	rmmod tpm
	img_part=/dev/nvme1n1p1
	img_path=/chromeos.img
	search --no-floppy --set=root --file $img_path
	loopback loop $img_path
	source (loop,12)/efi/boot/settings.cfg
	linux (loop,7)/kernel boot=local noresume noswap loglevel=7 options= chromeos_bootsplash= edit_brunch_config=1 \
		cros_secure cros_debug loop.max_part=16 img_part=$img_part img_path=$img_path
	initrd (loop,7)/lib/firmware/amd-ucode.img (loop,7)/lib/firmware/intel-ucode.img (loop,7)/initramfs.img
}
********************************************************************************

Notice that there are 2 entries... the top one is the main Chromeos entry & the 2nd entry is the configuration entry.  In this case, my ntfs partition is /dev/nvme1n1p1; yours would be /dev/sda5

So you would create 2 grub2win entries and copy the lines inside the {}'s
